lizard_connector/connector.py

```diff
@@ -0,0 +1,342 @@
+# coding=utf-8
+"""
+Connector to Lizard api (e.g. https://demo.lizard.net/api/v2) for python.
+
+Includes:
+- Endpoints (Lizard api endoints, e.g. TimeseriesEndpoint)
+- Fetcher (http handling)
+- queryfunctions for special cases such as geographical queries and time
```

Comment:
In principe is dit correct de hoofdletters voor Endpoints en Fetcher zijn om aan te duiden dat het classes zijn. Verder mag je bij een opsomming punten gebruiken, maar dat vind ik hier zelf niet zo logisch omdat het geen (of in ieder geval niet allemaal) zinnen zijn.